test/units/testsuite-74.networkctl.sh

```diff
@@ -80,6 +85,9 @@ networkctl cat @test2:network | cmp - <(networkctl cat "$NETWORK_NAME")
 EDITOR='cp' script -ec 'networkctl edit @test2 --drop-in test2.conf' /dev/null
 cmp "+4" "/etc/systemd/network/${NETWORK_NAME}.d/test2.conf"
 
+sleep 1
```

Comment:
So without the `sleep` here, the test seems to fail consistently. @yuwata Is there any chance that the reloading of networkd is not synchronous?

Reply:
Unfortunately, reloading is not synchronous.
https://github.com/systemd/systemd/blob/97f9a25cfd87b82feec1e67d2ffdf2be1aaa349e/test/test-network/systemd-networkd-tests.py#L738-L743

Reply:
I think this should be fixed in a future release.

Reply:
-> #31247.